backend/tests/data/conftest.py

```diff
@@ -1,7 +1,6 @@
-import pytest
-
 from unittest.mock import AsyncMock
 
+import pytest
```

Comment:
Why was this changed?

Reply:
That is the result of running the Trunk formatter.